sharktank_models/README.md

```diff
@@ -89,7 +89,7 @@ built as part of the [shark-ai project](https://github.com/nod-ai/shark-ai).
     pytest -n 4
     ```
 
-* Create an HTMl report using https://pytest-html.readthedocs.io/en/latest/index.html
+* Create an HTML report using https://pytest-html.readthedocs.io/en/latest/index.html
```

Comment:
Can you fix in this other file too? https://github.com/iree-org/iree-test-suites/blob/c47d13c0730a51beed3bef6128e7a61a80f85ce9/onnx_models/README.md?plain=1#L91

I also think it may be time soon to merge some of the docs and utility scripts for the test suites using pytest. I wanted to keep them standalone at the start, but it would be useful to be able to run multiple types of tests in a single invocation and have consistent options between all of them.

Reply:
Oh, I did not notice that because this is created when I'm reading the README.md. I verified that there are no `HTMl` in the repo with the fix. Done!

Reply:
Yeah, there's a fair bit of copy/paste at the moment, typos and all 😅